tests/torch/test_hybrid_converter.py

```diff
@@ -142,6 +142,30 @@ def test_gpt2_hybrid_mlp(list_or_str_private_modules_names, expected_accuracy, h
     prompt = "A long time ago,"
     input_ids = tokenizer.encode(prompt, return_tensors="pt")
 
+    # GPT2 has Conv1D layers with reshape. We need to convert them
+    # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/3858
+    def convert_conv1d_to_linear(module):
+        """Convert all Conv1D layers in a module to nn.Linear.
+
+        Args:
+            module (nn.Module): The module to be converted.
+
+        Returns:
+            nn.Module: The updated module with Conv1D layers converted to Linear layers.
+        """
+        for name, child in module.named_children():
+            if isinstance(child, Conv1D):
+                linear = torch.nn.Linear(child.weight.size(1), child.weight.size(0))
+                linear.weight.data = child.weight.T
+                if child.bias is not None:
+                    linear.bias.data = child.bias
+                setattr(module, name, linear)
+            else:
+                convert_conv1d_to_linear(child)
+        return module
+
+    convert_conv1d_to_linear(model)
```

Comment:
I don't think we should move this function

Reply:
Alright let's keep it for now I reverted.